pkg/new-ui/v1beta1/backend.go

```diff
@@ -37,8 +40,11 @@ func NewKatibUIHandler(dbManagerAddr string) *KatibUIHandler {
 		log.Printf("NewClient for Katib failed: %v", err)
 		panic(err)
 	}
+	sarclient := kclient.GetClient()
+
 	return &KatibUIHandler{
 		katibClient:   kclient,
+		sarClient:     sarclient,
 		dbManagerAddr: dbManagerAddr,
 	}
```

Comment:
Do you need `sarClient` parameter in `KatibUIHandler` ?
I think you can pass K8s client to `IsAuthorized` as follows:
```go
err=  IsAuthorized(user, "create", namespace, "kubeflow.org", "", "", experimentv1beta1.SchemeGroupVersion, k.GetClient())
```

Reply:
Valid point. Fixed!